changelog/comma-deprecation-error.dd

```diff
@@ -0,0 +1,4 @@
+The deprecation period of using the result of comma expression has ended
+
+Comma expressions have proven to be a frequent source of confusion, and bugs.
```

Comment:
Please include an example that now fails to compile and show how to correct it.

Reply:
Perhaps include an example showing how the comma expression can still be used without using the result.

Reply:
I was hoping I could avoid the redundancies from https://dlang.org/deprecate.html#Using%20the%20result%20of%20a%20comma%20expression, but okay copying doesn't cost us much.
I copied the text I have written for dlang.org a while ago.